test/unit/effects.js

```diff
@@ -13,20 +13,50 @@ var oldRaf = window.requestAnimationFrame,
 
 QUnit.module( "effects", {
 	setup: function() {
-		window.requestAnimationFrame = null;
 		this.sandbox = sinon.sandbox.create();
 		this.clock = this.sandbox.useFakeTimers( 505877050 );
+
 		this._oldInterval = jQuery.fx.interval;
+		this._oldPerformanceNow = window.performance.now;
 		jQuery.fx.step = {};
 		jQuery.fx.interval = 10;
+
+		// Simulate rAF using the mocked setTimeout as otherwise we couldn't test
+		// the rAF + performance.now code path.
+		// Support: IE <10, Android <4.4, Node with jsdom 9.4
+		if ( window.requestAnimationFrame ) {
+			window.requestAnimationFrame = function( callback ) {
```

Comment:
Good idea with sinon stubs, I've just switched to them.

As for the linked libs, I haven't checked them. We need something that will return values much smaller than `Date.now` to test that there is no divergence between various used timestamps.

The problem is I'd like it to hook to the same fake timer API that regular `Date.now()` does so that `this.clock.tick( number )` can be used in tests without changes for both code paths. Does it handle that?

My mock should be enough for our purposes, shouldn't it?